template.yml

```diff
@@ -36,8 +36,15 @@ Parameters:
     Type: String
     Default: "https://github.com"
     MinLength: 1
+  StageNameParameter:
+    Type: String
 
 Resources:
+  GithubOAuthApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: !Ref StageNameParameter
+      OpenApiVersion: "2.0"
```

Comment:
needed to get around the `sam` stage bug: https://github.com/awslabs/serverless-application-model/issues/191